extensions.json

```diff
@@ -818,23 +818,22 @@
     {
       "id": "ms-vscode.js-debug",
       "repository": "https://github.com/microsoft/vscode-js-debug",
-      "version": "1.50.0",
-      "checkout": "v1.50.0",
-      "prepublish": "npm run compile && npm run package",
-      "extensionFile": "dist/js-debug.vsix"
+      "version": "1.50.1",
+      "checkout": "master",
+      "prepublish": "npm run compile && npm run package"
     },
     {
       "id": "ms-vscode.node-debug",
       "repository": "https://github.com/microsoft/vscode-node-debug",
-      "version": "1.44.8",
-      "checkout": "v1.44.8",
+      "version": "1.44.14",
+      "checkout": "master",
```

Comment:
The latest release tag still seems to be `v1.44.8`. Please undo this change.

Reply:
Done!